test/framework/options.py

```diff
@@ -605,12 +605,13 @@ def test_list_easyblocks(self):
         self.eb_main(args, logfile=dummylogfn)
         logtxt = read_file(self.logfile)
 
-        for pat in [
-                    r"EasyBlock\s+\(easybuild.framework.easyblock\)\n",
-                    r"|--\s+EB_foo\s+\(easybuild.easyblocks.foo\)\n|\s+|--\s+EB_foofoo\s+\(easybuild.easyblocks.foofoo\)\n",
-                    r"|--\s+bar\s+\(easybuild.easyblocks.generic.bar\)\n",
-                   ]:
-
+        patterns = [
+            r"EasyBlock\s+\(easybuild.framework.easyblock\)\n",
+            r"\|--\s+EB_foo\s+\(easybuild.easyblocks.foo @ .*/sandbox/easybuild/easyblocks/f/foo.py\)\n" +
+            "\|\s+\|--\s+EB_foofoo\s+\(easybuild.easyblocks.foofoo @ .*/sandbox/easybuild/easyblocks/f/foofoo.py\)\n",
```

Comment:
are you missing an "r" here. It looks suspicious.

Reply:
I'll add it, but it doesn't matter in this context.